### PR TITLE
sidebar-toggling, hotkey

### PR DESCRIPTION
**Pull request by ghstng**

### CODE REVIEW

File: src/components/CoreInterface.vue [hunk 1]
```diff
@@ -36,7 +37,12 @@
           this.$refs.content
         ]
       },
-
+      keymap() {
```
Comment (rathboma) [inline src/components/CoreInterface.vue:40]: This should be in `data`

File: src/components/sidebar/CoreSidebar.vue [hunk 2]
```diff
@@ -2,7 +2,8 @@
 
   <div class="sidebar-wrap row">
     <global-sidebar
-      @click="click"
+      @selected="click"
+      v-on="$listeners"
```
Comment (rathboma) [inline src/components/sidebar/CoreSidebar.vue:6]: Nice, I've never used this, cool.

**Review by rathboma — CHANGES_REQUESTED** [review]
One minor thing - moving the keymap to `data`.

**Comment by rathboma** [comment]
One thing we've forgotten:

Adding a menubar entry for this under `view`. I should be able to do that before we merge.

**Comment by rathboma** [comment]
@ghstng it is now not possible to resize the sidebar at all.

**Comment by rathboma** [comment]
image

If you make the window smaller

**Review by rathboma — CHANGES_REQUESTED** [review]
Problems:

1. Can't resize the sidebar at all
2. It seems to start really wide
3. Make a small window and it squishes into the side

**Comment by rathboma** [comment]
I'm unfortunately going to undo your keymap too -- ctrl+b breaks standard mac and ubuntu keyboard shortcuts when editing (that's back a character in emacs-land, which both OSes bind to)

**Comment by ghstng** [comment]
@rathboma rgr will look.

**Pull request merged**